src/os/filesystem.go

```diff
@@ -57,6 +57,8 @@ type FileHandle interface {
 
 	// Close closes the file, making it unusable for further writes.
 	Close() (err error)
+
+	Fd() uintptr
```

Comment:
Instead of adding a new method to this interface (which breaks packages like tinyfs!), I would suggest type-asserting in the `Fd()` function.

Reply:
tinyfs was probably broken when we added the Seek() and ReadAt() methods to FileHandle, but yeah, avoiding further divergence would probably be good.

Reply:
See also https://github.com/tinygo-org/tinygo/issues/2709

Reply:
Tinyfs itself doesn't care about the discrepancy, its tests still pass because it has its own copy of File and FileHandle, and doesn't use tinygo's...?